README.md

```diff
@@ -1,58 +1,81 @@
-# ![nf-core](nf-core-logos/nf-core-logo.png)
+<img src="nf-core-logos/nf-core-logo.png" width="400">
 
-## Repository contents
+# [nf-core/logos](https://github.com/nf-core/logos)
+
+A repository for hosting files containing the official `nf-core` logo along with instructions on how to create an `nf-core` logo for your pipeline.
+
+## Table of contents
+* [Repository contents](#repository-contents)
+* [Create a pipeline logo](#create-a-pipeline-logo)
+  * [Docker](#docker)
+  * [Singularity](#singularity)
+  * [Locally](#locally)
+* [Correct logo](#correct-logo)
+* [Incorrect logo](#incorrect-logo)
+* [Help](#help)
 
-### [nf-core-logos/](nf-core-logos)
+## Repository contents
 
-- Contains original logo and icon in `ai` format, exported to `png` and also converted to `svg`
+* [nf-core-logos/](nf-core-logos) - contains official `nf-core` logo and icon in `ai`, `png` and `svg` formats.
+* [example-logos/](example-logos) - contains example pipeline logos in correct and incorrect rendering.
+* [make_logo/](make_logo) - contains a template logo with `GenericName` for a new pipeline, and a minimalist bash script to generate a new logo for your pipeline.
 
-### [make_logo/](make_logo)
+## Create a pipeline logo
 
-- Contains a template logo with `GenericName` for a new pipeline, and a minimalist bash script to generate a new logo
+There are various ways you can create a logo for your pipeline. Each of these methods will generate the logo in `png` and `svg` formats e.g. `NewPipeline_logo.png` and `NewPipeline_logo.svg`
 
-## Create logo using Docker
+### Docker
 
-We provide a Docker Image on [DockerHub](https://cloud.docker.com/u/nfcore/repository/docker/nfcore/logos) that you may use to create logos for your pipeline easily.
+We provide a Docker Image on [DockerHub](https://cloud.docker.com/u/nfcore/repository/docker/nfcore/logos) that you may use to create logos for your pipeline easily:
 
 ```bash
-docker run -v /path/on/host:/data -it nfcore/logos bash
-/make_logo.sh NewPipeline
-mv NewPipeline* /data
+docker run -v `pwd`:`pwd` -w `pwd` nfcore/logos NewPipeline
```

Comment:
@apeltzer Ive add this in as suggested by @ewels but I have no idea how to implement `ENTRYPOINT` in the `Dockerfile`. Can you send me a file/code-block with the changes, and ill push it.

Reply:
see eg. [here](https://github.com/ewels/MultiQC/blob/39d83c2e5991f8911e7f522987a1c8f5851c9c5b/Dockerfile#L22) - basically exactly as in my [comment](https://github.com/nf-core/logos/pull/7#discussion_r282467198).. 😀

Reply:
BOOOOM! Like now?

Reply:
PS: Im registering for ISMB. I dont have time to learn `Docker` right now.

Reply:
almost - I don’t think you need the `CMD` though. That’ll make a logo with the title `.` if a name is not specified...